src/app/campaign/inventory/inventory-table.component.ts

```diff
@@ -0,0 +1,122 @@
+import { Component, ChangeDetectionStrategy, Input } from '@angular/core';
+import { Flight, FlightDay, InventoryRollup, InventoryWeeklyRollup } from '../store/models';
+import { getMidnightUTC, getDateSlice } from '../store/selectors';
+
```

Comment:
Extracted from the previously named availability.component and renamed to inventory-table. I felt like this table part needed to be its own component.